### DISCUSSION TITLE
ERROR 1: PROJ: proj.db contains DATABASE.LAYOUT.VERSION.MINOR = 0 whereas a number >= 2 is expected.

### DISCUSSION THREAD

**Discussion by menavarroc** [pyproj4/pyproj#1045] [Q&A] [answered]
Hi every one,

I am getting the next Error message when I run a python process 

`ERROR 1: PROJ: proj_create_from_database: C:\Program Files\PostgreSQL\13\share\contrib\postgis-3.2\proj\proj.db contains DATABASE.LAYOUT.VERSION.MINOR = 0 whereas a number >= 2 is expected. It comes from another PROJ installation.
`

I installed pyproj from pypi and the code where I tranform a coordinates from utm to gcs is this.

inProj = CRS.from_epsg(int(f"326{zona}"))    
outProj = CRS.from_epsg(4326)    
transformer = Transformer.from_crs(inProj, outProj)

Anybody knows how to solve this issue?

os: windows 10
python 3.9
pyporj version: 3.3.0, 
postgres 13
postgis version: 3.2

**Comment by snowman2** [comment, marked as answer]
You are have multiple versions of PROJ installed and the incorrect database is being found. Here are debugging suggestions: https://pyproj4.github.io/pyproj/stable/gotchas.html#internal-proj-error-sqlite-error-on-select

**Comment by alberto-dri** [comment]
Please set the environment variable PROJ_LIB to the right database (or directory).  For example:

$ echo $PROJ_LIB
C:\Program Files\PostgreSQL\14\share\contrib\postgis-3.2\proj        << This is another database installed in my computer

My most recent installation of pyproj exists in C:/myprojects/virtualenvs/p310/Lib/site-packages/pyproj.

Therefore I must set PROJ_LIB to:
$ export PROJ_LIB=C:/myprojects/virtualenvs/p310/Lib/site-packages/pyproj/proj_dir/share/proj

I hope this helps.

**Reply by lagamura** [reply]
Thanks for the detailed comment, I am facing the same issue in a GDAL container (latest 3.11) with `pyproj==3.7.1`.
Why in your case the pyproj path preceded the system's one?

I am seeing also other relative env vars mentioned like $PROJ_DATA.

*EDIT: I found the $PROJ_LIB is an alias to $PROJ_DATA: https://proj.org/en/stable/usage/environmentvars.html#envvar-PROJ_DATA

**Comment by frederikvand** [comment]
\Library\share\proj\proj.db contains DATABASE.LAYOUT.VERSION.MINOR = 2 whereas a number >= 4 is expected. It comes from another PROJ installation.

I got the same error after updating a package (xarray). Even with a clean install of conda and my environment (which works on a different computer) the issue keeps recurring. Even when uninstalling all other geo related software the problem remains. The only solution I have left is wiping my computer and reinstalling my OS.
